backend/tests/test_portland_ingest.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+import pytest
+
+
+@pytest.fixture
+def dataset():
+
+    file_path = "/Users/jimenasalinas/Library/CloudStorage/Box-Box/Route Rangers/Transit dataset exploration/Portland Ridership Data/portland_ridership/PDX_ridership.json"
```

Comment:
probably don't want local filepaths for testing because it will break for folks that don't have that route, if you are using json maybe you can mock a part of the data in a dictionary?